src/parthenon_manager.cpp

```diff
@@ -336,6 +336,9 @@ void ParthenonManager::RestartPackages(Mesh &rm, RestartReader &resfile) {
 
       // Double note that this also needs to be update in case
       // we update the HDF5 infrastructure!
+      // JMM: Version -1 is the old version. We are now on version 3.
+      // TODO(JMM): Do we need to maintain backwards compatibility
+      // here? It would be ncie to remove this.
```

Comment:
I wouldn't mind to remove the old stuff (potentially in separate PR as it'd also pertain to the phdf reader) and mark is as deprecated.

Reply:
:+1: I don't see a way to add the `[[deprecated]]` atribute to arbitrary scope. I think it needs to be added to a function, variable or class definition. But I added a `PARTHENON_WARN`.